packages/istanbul-lib-source-maps/lib/transformer.js

```diff
@@ -148,141 +151,138 @@ function getMapping(sourceMap, generatedLocation, origFile) {
     };
 }
 
-function SourceMapTransformer(finder, opts) {
-    opts = opts || {};
-    this.finder = finder;
-    this.baseDir = opts.baseDir || process.cwd();
+function getUniqueKey(path) {
```

Comment:
nitpick: this is sort of a big file, is there any separation we can do? e.g. moving out utility methods or at least moving them to the bottom? My personal preference is having the thing that the file is named after exported at the very top (with the `export default` keywords) but that may be N/A here.

Reply:
I've moved to a couple separate files.  Personally my preference is against hoisting, so declaring a function before it is used.  Pulling from a separate file is fine as well.